monkey/common/agent_configuration/default_agent_configuration.py

```diff
@@ -17,7 +19,7 @@
 
 PAYLOAD_CONFIGURATION: Dict[str, Dict] = {}
 
-TCP_PORTS = (
+TCP_PORTS: Tuple[int, ...] = (
```

Comment:
Just make this `Tuple[NetworkPort, ...]` instead of converting it everywhere it's used. Same with `HTTP_PORTS`.